lib/puppet/util/windows/principal.rb

```diff
@@ -34,7 +34,7 @@ def ==(compare)
 
     # added for backward compatibility
     def to_s
-      @sid
+      @domain_account
```

Comment:
nit: not sure if we need to do this...

Reply:
It's required for `puppet resource` to properly emit manifests